src/Protector.php

```diff
@@ -667,4 +667,26 @@ public function setServerUrl(string $serverUrl): void
     {
         $this->serverUrl = $serverUrl;
     }
+
+    /**
+     * Returns the name of the most recent dump.
+     *
+     * @return string
+     */
+    public function getLatestDumpName(): string
+    {
+        $disk          = $this->getDisk();
+        $baseDirectory = $this->getConfigValueForKey('baseDirectory');
+        $files         = $disk->files($baseDirectory);
```

Comment:
Should we also filter the files by their `extension()` to prevent files like `.gitignore` or whatever to be accidently selected?

Reply:
The file name can be customized by the user in the config, therefore the dump files might have some crazy extensions, since mysql doesn't really care about it when importing the dump.